editor.planx.uk/src/@planx/components/Checklist/Public.tsx

```diff
@@ -44,6 +46,15 @@ function getFlatOptions({
   return [];
 }
 
+const FullWidthContainer = styled(Box)(({ theme }) => ({
+  ...fullWidthContent(theme),
+}));
+
+const FormWrapper = styled(Box)(({ theme }) => ({
+  width: theme.breakpoints.values.formWrap,
+  maxWidth: "100%",
+}));
```

Comment:
These two components and their styles are repeated a few time across various files. There a are a few ways we could reduce this - 

- Make a variant (e.g. `<Container variant="formWrapper">`)
- Make a standalone UI component (e.g. `<FormWrapper>`)

I don't really think there's a compelling difference between the two - it's much more about semantics and what's clearer when reading the code. Generally we're favouring standalone components over variants and I think that's probably the only deciding factor here. It's worth having a look through [the docs](https://mui.com/material-ui/customization/theme-components/#creating-new-component-variants) about variants to get a good understanding of the concept though.

Here's how I'd approach making a standalone UI component here - 

 - Make new components for both of these in `src/ui` directory (there should be a few simple examples here)
 - Import these components into the files where they're needed
 
If this isn't clear lets jump on a call and do some pairing on it 👨‍💻 👨‍💻

Reply:
I had looked into using variants for `<Container>` before and MUI doesn't support variants for this, so a standalone UI seems like the best route. I've had a go at setting one up based on the other examples, seems to work as expected.

Reply:
Ah yeah - not all components allow variants - that's a bit of an annoying restriction but it's not a real issue here!